tools/test/examples/examples.json

```diff
@@ -37,18 +37,20 @@
       "name": "mbed-os-example-ble",
       "github":"https://github.com/ARMmbed/mbed-os-example-ble",
       "mbed": [
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-Beacon",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-HeartRate",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-Thermometer",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-LEDBlinker",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-LED",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-GAPButton",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-BatteryLevel",
           "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-Button",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-BatteryLevel"
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-GAP",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-GAPButton",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-GattClient",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-GattServer",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-HeartRate",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-privacy",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-SM",
+          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-ble-Thermometer"
       ],
       "test-repo-source": "mbed",
       "features" : ["BLE"],
-      "targets" : ["NRF51_DK", "NRF52_DK", "K66F", "NUCLEO_F401RE"],
```

Comment:
K66F was to move away from K64F, to make building faster. test team will review

Reply:
https://github.com/ARMmbed/mbed-os/pull/8835

Reply:
Thanks - I now understand what is needed - will remove the 66/64 since it is identical to the NRF51 configuration which uses all defaults, and will tweak farther for new targets we have had build issues with before.